eZ/Publish/Core/Persistence/Legacy/Content/UrlWildcard/Gateway/DoctrineDatabase.php

```diff
@@ -162,4 +162,33 @@ public function loadUrlWildcardsData($offset = 0, $limit = -1)
 
         return $stmt->fetchAll(\PDO::FETCH_ASSOC);
     }
+
+    /**
+     * Loads the UrlWildcard with given $url.
+     *
+     * @param string $sourceUrl
+     *
+     * @return array
+     */
+    public function loadUrlWildcardBySourceUrl(string $sourceUrl)
+    {
+        $query = $this->dbHandler->createSelectQuery();
```

Comment:
Please use `Doctrine\Connection` to build queries, this will be gone in the next major (if time allows :P)

Reply:
No, it's not possible. I would have to:
a) refactor the whole class to use `Doctrine\Connection`, which is far away from the scope of this PR,
b) have to inject both `\eZ\Publish\Core\Persistence\Database\DatabaseHandler` and `Doctrine\Connection` in the same time. I'm pretty sure that you wouldn't accept this approach afterward 😉

Reply:
> No, it's not possible. I would have to:
> a) refactor the whole class to use `Doctrine\Connection`, which is far away from the scope of this PR,
> b) have to inject both `\eZ\Publish\Core\Persistence\Database\DatabaseHandler` and `Doctrine\Connection` in the same time. I'm pretty sure that you won't accept this approach afterward

@kmadejski DatabaseHandler will be gone in 3.0/8.0, so this will stop working. I'm not accepting technical debt at this point. Please see how you can achieve something in the middle - using Connection, but not injecting it and not refactoring changes out of scope. It is possible.

Reply:
@alongosz done in https://github.com/ezsystems/ezpublish-kernel/pull/2609/commits/caaa05b32e11188af3ff5e91ce23845b8ad3ccdd.